cmd/erasure-sets.go

```diff
@@ -629,6 +629,10 @@ func (s *erasureSets) MakeBucketWithLocation(ctx context.Context, bucket string,
 	return nil
 }
 
+func idK0K1(id []byte) (k0, k1 uint64) {
```

Comment:
Can we give the function a more descriptive name - e.g. `splitInt128` since that is what it does:
Splitting an 128bit value into two 64bit values.

Reply:
It could also be manually "inlined" with no read downsides.

Reply:
Sure @aead @klauspost